src/framework/preferences/js/Panels.js

```diff
@@ -150,8 +150,11 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         // in order to reset the dom binder when panels are in an iframe.
         // It can be be eliminated once we have the new renderer.
         var userJQuery = that.container.constructor;
-        var oldContainer = that.container;
-        that.container = userJQuery(that.container.selector, oldContainer[0].ownerDocument);
+        var context = that.container[0].ownerDocument;
+        var selector = that.container.selector;
+        that.container = userJQuery(selector, context);
```

Comment:
This specific JIRA doesn't exist. I suppose we could file something after the fact if that's what you mean. Also, in the end I'm not sure this was specific to the iframe boundary but rather a product of recreating the container inside resetDomBinder method. Which is why the changes were needed here as well. I left the changes in `fluid.container` mostly to guard against the case where someone does something like `fluid.someComponent($(".someContainer"), {});`